### PR TITLE
fix(backup): modify backup volume and backup backing image name

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue # longhorn/longhorn#5411

### CODE REVIEW

File: types/types.go [hunk 1]
```diff
@@ -738,15 +741,11 @@ func GetShareManagerNameFromShareManagerPodName(podName string) string {
 }
 
 func GetBackupVolumeNameFromVolumeName(volumeName string) string {
-	return generateBackupResourceName(volumeName)
-}
-
-func generateBackupResourceName(resourceName string) string {
-	return util.GetStringChecksum(strings.TrimSpace(resourceName))[:util.RandomIDLength] + "-" + util.RandomID()
+	return volumeName + backupVolumeSuffix + "-" + util.RandomID()
```
Comment (derekbit) [inline types/types.go:744]: `backupVolumeSuffix` seems not really required?

File: types/types.go [hunk 2]
```diff
@@ -738,15 +741,11 @@ func GetShareManagerNameFromShareManagerPodName(podName string) string {
...
 }
 
 func GetBackupBackingImageNameFromBIName(backingImageName string) string {
-	return generateBackupResourceName(backingImageName)
+	return backingImageName + backupBackingImageSuffix + "-" + util.RandomID()
```
Comment (derekbit) [inline types/types.go:748]: `backupBackingImageSuffix` seems not really required?

File: types/types.go (hunk 1 above)
Comment (mantissahz) [reply types/types.go:744]: Removed. Thanks

**Review by derekbit — APPROVED** [review]
LGTM

**Pull request merged**